timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -0,0 +1,54 @@
+types:
+  imports:
+    LeaderTime:
+      base-type: any
+      external:
+        java: com.palantir.lock.v2.LeaderTime
+    NanoTime:
+      base-type: safelong
+      external:
+        java: com.palantir.common.time.NanoTime
+    PartitionedTimestamps:
+      base-type: any
+      external:
+        java: com.palantir.lock.v2.PartitionedTimestamps
+    LockImmutableTimestampResponse:
+      base-type: any
+      external:
+        java: com.palantir.lock.v2.LockImmutableTimestampResponse
+    Lease:
+      base-type: any
+      external:
+        java: com.palantir.lock.v2.Lease
+  definitions:
+    default-package: com.palantir.atlasdb.timelock.api
+    objects:
+      ConjureStartTransactionsRequest:
+        fields:
+          requestId: uuid
+          requestorId: uuid
+          numTransactions: integer
```

Comment:
@gmaretic how easily can we retrofit lock watch pieces here? Can we do it afterwards without adding startTransactionsV2?

Reply:
I think I can answer this: yes, you can just add the lock watches to be registered / lock watch state to the request/response type respectively.